### PR TITLE
Consider properties for equality of ObjectProperty

### PR DESCRIPTION
**Pull request by mduesterhoeft**
Addresses https://github.com/swagger-api/swagger-core/issues/3084

### CODE REVIEW

**Comment by mduesterhoeft** [comment]
Hi there, what do you think? Does this have a chance to be merged soon? @frantuma

File: modules/swagger-models/src/main/java/io/swagger/models/properties/ObjectProperty.java [hunk 1]
```diff
@@ -139,4 +139,25 @@ public ObjectProperty example(Object example) {
         this.setExample(example);
         return this;
     }
+
+    @Override
+    public boolean equals(Object o) {
```
Comment (frantuma) [inline modules/swagger-models/src/main/java/io/swagger/models/properties/ObjectProperty.java]: can you possibly just add
```
if (this == o) {
  return true;
}
```

**Comment by frantuma** [comment]
Thanks!

**Comment by mduesterhoeft** [comment]
Thanks for merging.

**Comment by mduesterhoeft** [comment]
@frantuma do you already have plans to release the next 1.5 version?

**Comment by frantuma** [comment]
It will be within 1-2 weeks

**Pull request merged**